lib/Mojo/Util.pm

```diff
@@ -540,7 +563,7 @@ Mojo::Util - Portable utility functions
 
 =head1 SYNOPSIS
 
-  use Mojo::Util qw(b64_encode url_escape url_unescape);
+  use Mojo::Util go(b64_encode url_escape url_unescape);
 
```

Comment:
Unintended change of the quote operator?

Reply:
Fixed